### PR TITLE
fix: trending drawer overflow

### PR DESCRIPTION
**Pull request by kovsu**
https://github.com/RSSNext/Follow/commit/3edee63a5a9f691481c8532ba524d541ca09c539#diff-2bd15723d33f49ade6aaf527ddfd0549ae37e38e6c5d456e4b88cb6da91c3415R142

This code seems lost. It makes the below problem.

image

### CODE REVIEW

File: apps/renderer/src/components/ui/scroll-area/ScrollArea.tsx [hunk 1]
```diff
@@ -137,7 +137,10 @@ export const ScrollArea = React.forwardRef<
         <Viewport
           ref={setViewportRef}
           onWheel={stopPropagation}
-          className={cn(flex ? "[&>div]:!flex [&>div]:!flex-col" : "", viewportClassName)}
+          className={cn(
+            flex ? "[&>div]:!flex [&>div]:!flex-col" : "[&>div]:!block",
```
Comment (Innei) [inline apps/renderer/src/components/ui/scroll-area/ScrollArea.tsx:141]: You can't use `block`, it will cause an exception in the radix scrollbar.

**Review by Innei — CHANGES_REQUESTED** [review]

File: apps/renderer/src/components/ui/scroll-area/ScrollArea.tsx (hunk 1 above)
Comment (kovsu) [reply apps/renderer/src/components/ui/scroll-area/ScrollArea.tsx]: What's the problem? Can you point out its details? 🥹

**Comment by kovsu** [comment]
I think it would work fine here if I added this class to the trending drawer.

https://github.com/user-attachments/assets/69e6f301-ef22-4565-84aa-6d8c950c0ac7

**Review by Innei — APPROVED** [review]
It makes sense to modify only the class of this component, not the generic component.

**Pull request merged**